src/main/java/digital/slovensko/autogram/ui/gui/GUI.java

```diff
@@ -199,6 +199,34 @@ public void showPkcsEidWindowsDllError(AutogramException e) {
         stage.show();
     }
 
+    public char[] getDocumentPassword() {
+        var futurePassword = new FutureTask<>(() -> {
+            var controller = new PasswordController("Aké je heslo k dokumentu?", "Zadajte kód k dokumentu.", false, false);
+            var root = GUIUtils.loadFXML(controller, "password-dialog.fxml");
+
+            var stage = new Stage();
+            stage.setTitle("Odomknutie dokumentu");
```

Comment:
Kebyže tu niekam dostaneme názov dokumentu, bolo by to super. Napr. pri hromadnom neviem, o ktorý dokument ide